schutzbot/sonarqube.sh

```diff
@@ -0,0 +1,27 @@
+#!/bin/bash
+
+set -euxo pipefail
+
+export SONAR_SCANNER_OPTS="-Djavax.net.ssl.trustStore=schutzbot/RH-IT-Root-CA.keystore -Djavax.net.ssl.trustStorePassword=$KEYSTORE_PASS"
+sudo dnf install -y unzip
+curl "https://binaries.sonarsource.com/Distribution/sonar-scanner-cli/sonar-scanner-cli-$SONAR_SCANNER_CLI_VERSION-linux.zip" -o sonar-scanner-cli.zip
+unzip -q sonar-scanner-cli.zip
+
+SONAR_SCANNER_CMD="sonar-scanner-$SONAR_SCANNER_CLI_VERSION-linux/bin/sonar-scanner"
+
+$SONAR_SCANNER_CMD -Dsonar.projectKey=osbuild-github \
+                   -Dsonar.sources=. \
+                   -Dsonar.host.url=https://sonarqube.corp.redhat.com \
+                   -Dsonar.login="$SONAR_SCANNER_TOKEN" \
+                   -Dsonar.pullrequest.branch="$CI_COMMIT_BRANCH" \
+                   -Dsonar.pullrequest.key="$CI_COMMIT_SHA" \
+                   -Dsonar.pullrequest.base="main"
+
+SONARQUBE_URL="https://sonarqube.corp.redhat.com/dashboard?branch=$CI_COMMIT_BRANCH&id=osbuild-github"
+# Report back to GitHub
```

Comment:
This will report a successful completion of the CI job, iow sending the scanner artifacts to SonarQube. 

Question: Will the hosted SQ instance report back to GitHub with its findings or send some emails at least ?

Reply:
> Question: Will the hosted SQ instance report back to GitHub with its findings or send some emails at least ?

I plan to do that. Since I don't know how long it takes for the results to be ready I was thinking of utilizing webhooks to check when the result are ready and report that in the custom status here on Github.